pkg/extensions/config/sync/config.go

```diff
@@ -33,7 +33,8 @@ type RegistryConfig struct {
 	RetryDelay       *time.Duration
 	OnlySigned       *bool
 	CredentialHelper string
-	PreserveDigest   bool // sync without converting
+	PreserveDigest   bool     // sync without converting
+	Architectures    []string // filter architectures during sync
```

Comment:
https://github.com/opencontainers/image-spec/blob/main/image-index.md#platform-variants

To be more accurate, the following:
```
"platform": {
        "architecture": "ppc64le",
        "os": "linux"
      }
  ```

Reply:
Added in https://github.com/project-zot/zot/pull/3128/commits/0ab2c7a61360502bcfca13638f5a7a8e8f60b271